### DISCUSSION TITLE
Is vulnerabilityAlerts limited by prConcurrentLimit when creating PRs?

### DISCUSSION THREAD

**Discussion by richard-li-next** [renovatebot/renovate#16336] [Request Help]
I limit the number of automatically created PRs in the repo via prConcurrentLimit. But I don't know if dependabot alerts reports a vulnerability when the automatically created PR has reached the upper limit, then whether renovate will also create the corresponding PR

renovate.json
```json
{
  "extends": ["config:base", "schedule:earlyMondays", "npm:unpublishSafe"],
  "prConcurrentLimit": 2,
  "includePaths": ["+(package.json)"]
}
```

**Comment by rarkins** [comment]
Limits should be bypassed/ignored for vulnerabilities

**Comment by richard-li-next** [comment]
@rarkins How to bypass restrictions? is that so?
```
"vulnerabilityAlerts": {
    "prConcurrentLimit": 0
  }
```

**Reply by rarkins** [reply]
It should already be set. If you're not seeing this behaviour then paste a log here for the relevant branch
